katas/content/multi_qubit_gates/index.md

```diff
@@ -627,6 +644,15 @@ The sequence of steps that implement this variant are:
 2. Apply the regular controlled version of the gate.
 3. Apply the $X$ gate to the same qubits to return them to their original state.
 
+@[exercise]({
```

Comment:
Please move this exercise up to the Multi-Controlled Gates lesson, which is the one where we discuss Toffoli gate